tests/default_vars.sh

```diff
@@ -24,8 +24,11 @@
   TASKS_cpl_thrd=120; THRD_cpl_thrd=2
```

Comment:
The THRD setting will be used for just the ATM, right? Could we rename it to be ATM_cpl_thrd to match OCN_thrds, ICE_thrds etc? I think we'd need to change all the current THRD variables to be ATM_THRD and adjust the scripts.